### PR TITLE
PP-68-items-I-sell

### PR DESCRIPTION
**Pull request by Victoria240**
The data on this page has been hard-coded for now. I'll update the component when the API/route has been set up.

### CODE REVIEW

File: app/seller-items/page.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+"use client";
```
Comment (AnnaPestova1) [inline app/seller-items/page.js]: Hi! we have a route for this page 
image
/sell

File: app/seller-items/page.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                    <Typography variant="h6" gutterBottom mb={theme.spacing(2)}>
+                        Items Sold
+                    </Typography>
+                    <Button variant="outlined" color="primary" onClick={() => router.push("/sell")}>
+                        Add New Card
+                    </Button>
+                </Box>
+                <Grid container spacing={3}>
+                    {sellerItems.map(item => (
+                        <Grid item xs={12} sm={6} md={4} lg={3} key={item.id}>
```
Comment (AnnaPestova1) [inline app/seller-items/page.js]: What do you think if we will reduce the air between cards? Right now with current card sizes, I have a large gap between cards on the medium size screen 
image
and on the small size screen too 
image


 (I am not sure about this part)

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Victoria! I like how your page looks. There are some issues:  this part 
image
is from the user profile page. On the profile page, our user will see all information about himself. As a seller too. We do not need this pars here.

But on this page user should be able to add a new card (we have API for it),  edit a card and delete a card (edit and delete functionality is already in our API. We will have API to GET seller's cards in my next pull request. 

This 
image
 will be on the Profile page too. 

the add new card button doesn't have any margins:

image


Maybe you should look at the Container component https://mui.com/material-ui/react-container/. It will help you with the layout. 

if we add sold items, we also need to add available items.
image

I think the purpose of this page - is to allow user to see his cards, add, edit and delete them

File: app/market/item/[id]/page.jsx [hunk 3]
```diff
@@ -1,43 +1,136 @@
...
+            Home
+          </Link>
+          <Typography color="text.primary">Card Details</Typography>
+        </Breadcrumbs>
+
+        {/* Image and Details Section */}
+        <div style={{ display: 'flex', marginTop: theme.spacing(2)}}>
+          {/* Image Section */}
+          {cardDetails && (
+            <Card style={{ boxShadow: 'none', marginRight: theme.spacing(2) }}>
```
Comment (AnnaPestova1) [inline app/market/item/[id]/page.jsx:70]: we should use 
our CardComponent. its height will be fixed in the Luba's pull request.

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]

**Review by AnnaPestova1 — APPROVED** [review]
Great job, Victoria!

**Review by AnnaSolovykh — APPROVED** [review]

**Pull request merged**